R/drive_list.R

```diff
@@ -0,0 +1,162 @@
+#' List files on Google Drive
+#'
+#' @param path character vector, path name(s) where the Google drive files are
```

Comment:
This function won't be vectorized over `path`, so I would delete the `(s)` to avoid any confusion. If someone needs to do this for several paths, they'll need to map.